### PR TITLE
fix: message content text alignment and spacing

### LINKED ISSUE #17605: Design Feedback Communities: Message content text alignment and spacing
## Problem 
[comment]: # (A problem description)
Message text alignment and spacing doesn't match to Figma design
Design Feedback: https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=6117:85237&mode=design#570474775

#### Expected behavior
https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=6166-62737&mode=design&t=R6v2Aqc1s3n8aahu-4

#### Actual behavior
https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=6117:85237&mode=design#570474775

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by codemaster115**
fixes #17605 

### Summary
Resolve design feedback(1.25) to fix spacing and text alignment in message content
Design feedback: https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=6117:85237&mode=design#570474775

#### Platforms

- Android
- iOS
- macOS
- Linux
- Windows

### Before and after screenshots comparison
simulator_screenshot_CAF8EA37-485D-43A5-8F49-1621DDBA738D

status: ready

### CODE REVIEW

File: src/status_im2/contexts/chat/messages/content/text/style.cljs [hunk 1]
```diff
@@ -26,7 +26,8 @@
                                                colors/primary-60)
    :selection-color       :transparent
    :suppress-highlighting true
-   :line-height           21})
+   ;;  :line-height           21
```
Comment (ibrkhalil) [inline src/status_im2/contexts/chat/messages/content/text/style.cljs]: Please delete unused code.

**Review by ibrkhalil — APPROVED** [review]

File: src/status_im2/contexts/chat/messages/content/text/view.cljs [hunk 2]
```diff
@@ -104,10 +104,11 @@
...
                 (render-inline acc e chat-id style-override mention-first))
               [quo/text
-               {:style {:size          :paragraph-1
-                        :margin-bottom (if mention-first (if platform/ios? 4 0) 2)
+               {:size  :paragraph-1
+                :style {:margin-bottom (if mention-first (if platform/ios? 4 0) 2)
                         :margin-top    (if mention-first (if platform/ios? -4 0) -1)
-                        :color         (when (seq style-override) colors/white)}}]
+                        :color         (when (seq style-override) colors/white)
+                        :line-height   22.75}}]
```
Comment (flexsurfer) [inline src/status_im2/contexts/chat/messages/content/text/view.cljs:111]: could you please elaborate why we have custom line height here ?

**Comment by churik** [comment]
@codemaster115 
the icon  next to the display name looks completely misaligned (right side nightly build, left side - PR)
photo_2023-10-11 14 17 56 jpeg 2023-10-11 14-19-27
I suppose this should be fixed here to match the design from https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS?node-id=6117:85237&mode=design#570474775

Tested on IPhone 14 Pro.

File: src/status_im2/contexts/chat/messages/content/text/view.cljs (hunk 2 above)
Comment (codemaster115) [reply src/status_im2/contexts/chat/messages/content/text/view.cljs:111]: From figma design, I calculated the line height.

Comment (ajayesivan) [reply src/status_im2/contexts/chat/messages/content/text/view.cljs:111]: If I'm looking at the correct [figma design](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=6166-62737&mode=design&t=GbZ3g1O5BJCSq9ov-4), the `line-height` is `21.75` which is the exact `line-height` of the [`:paragraph-1`](https://github.com/status-im/status-mobile/blob/0a8993bbf19fc368e4f3f04ed5daf73062712203/src/quo2/foundations/typography.cljs#L29) text.

I also noticed that Figma uses [Message Type](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=3188-49964&mode=design&t=BwbRgofwzOmOkKrp-4) component from the design system that we haven't implemented in code yet.

**Comment by VolodLytvynenko** [comment]
Hi, @codemaster115 could you please rebase and resolve conflicts in the current PR? After that, we can perform manual testing. Thanx!

**Comment by VolodLytvynenko** [comment]
@codemaster115 Thank you for PR. No issues from my side

@Francesca-G could you take a look current PR? Thanx!

**Review by flexsurfer — APPROVED** [review]

**Review by Francesca-G — APPROVED** [review]
Looks fixed ✨

**Pull request merged**